ggml.c

```diff
@@ -247,7 +247,11 @@ inline static void* ggml_aligned_malloc(size_t size) {
 #include "ggml-opencl.h"
 #endif
 #elif defined(GGML_USE_OPENBLAS)
+#if defined(GGML_BLAS_USE_MKL)
+#include "mkl.h"
```

Comment:
Probably want angle brackets (`<mkl.h>`) to indicate a system library?

Reply:
@evanmiller thx for the feedback. I've changed the code.